_posts/2022-02-14-changelog-165.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #165"
+date: 2022-02-14 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8473] Provide support for 2022.1 EAPs
+
+* [#8456] Drop support of 2021.2 major IDE release. To get new updates of the plugin, upgrade your IDE to 2021.3
+
+* [#8455] Report misplaced lifetimes in `Wrong generic arguments order` [E0747](https://doc.rust-lang.org/error-index.html#E0747) inspection
+  
+  <img src="/assets/posts/changelog-165/lifetime_order.png" width="700px"/>
+
+* [#8266] Implement `Add unambiguous imports on the fly` option. You can turn it on in `Settings | Editor | General | Auto Import`. The option may be useful when pasting Rust code copied from outside the editor
+
+  {% include gif-img.html path="/assets/posts/changelog-165/unambiguous_imports" w="700px" %}
```

Comment:
Are you sure the gif provides a good example of the feature?
Maybe it's better to show something similar to gif in https://github.com/intellij-rust/intellij-rust/pull/8266 ?

https://plugins.jetbrains.com/plugin/7345-presentation-assistant plugin also can help you to show that you make copy action